ui/raidboss/data/07-dt/trial/byakko-un.ts

```diff
@@ -0,0 +1,421 @@
+// This file was autogenerated from running npm run sync-files.
+// DO NOT EDIT THIS FILE DIRECTLY.
+// Edit the source file below and then run `npm run sync-files`
+// Source: ui/raidboss/data/04-sb/trial/byakko-ex.ts
+
+import Conditions from '../../../../../resources/conditions';
+import { Responses } from '../../../../../resources/responses';
+import ZoneId from '../../../../../resources/zone_id';
+import { RaidbossData } from '../../../../../types/data';
+import { TriggerSet } from '../../../../../types/trigger';
+
+export interface Data extends RaidbossData {
+  roarCount?: number;
+  stakeCount?: number;
+}
+
+// Byakko Unreal
+const triggerSet: TriggerSet<Data> = {
+  id: 'TheJadeStoaUnreal',
+  zoneId: ZoneId.TheJadeStoaUnreal,
+  timelineFile: 'byakko-un.txt',
+  triggers: [
+    {
+      id: 'ByakkoUn Heavenly Strike',
+      type: 'StartsUsing',
+      netRegex: { id: '9BFB', source: 'Byakko' },
+      response: Responses.tankCleave(),
+    },
+    {
+      id: 'ByakkoUn Sweep The Leg Intermission', // Donut AoE, no outer safe spots
+      type: 'StartsUsing',
+      netRegex: { id: '9C15', source: 'Byakko', capture: false },
+      response: Responses.getIn(),
+    },
+    {
+      id: 'ByakkoUn Sweep The Leg Standard', // 270-degree frontal cleave
+      type: 'StartsUsing',
+      netRegex: { id: '9BFC', source: 'Byakko', capture: false },
+      response: Responses.getBehind(),
+    },
+    {
+      id: 'ByakkoUn Storm Pulse',
+      type: 'StartsUsing',
+      netRegex: { id: '9BFD', source: 'Byakko', capture: false },
+      response: Responses.aoe(),
+    },
+    {
+      id: 'ByakkoUn Distant Clap',
+      type: 'StartsUsing',
+      netRegex: { id: '9BFE', source: 'Byakko', target: 'Byakko', capture: false },
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Distant Clap',
+          de: 'Donnergrollen',
+          fr: 'Tonnerre lointain',
+          ja: '遠雷',
+          cn: '月环',
+          ko: '원뢰',
+        },
+      },
```

Comment:
```suggestion
      response: Responses.getUnder(),
```
We should normalize this call to current standards.

Reply:
Yes, you're right. I left it as it was because it's technically a donut that can be outranged, but it would probably be best to just standardize it.